o2a/templates/mapreduce.tpl

```diff
@@ -28,7 +28,7 @@
         archives={{ hdfs_archives | to_python }},
     {% endif %}
     cluster_name=CONFIG['dataproc_cluster'],
-    dataproc_hadoop_properties={% include "props.tpl" %},
+    dataproc_hadoop_properties={% include "xml_escaped_props.tpl" %},
```

Comment:
If the props still exists then I think it's better to use the macro.
http://jinja.pocoo.org/docs/2.10/templates/#macros
This is very similar to the templates, but it is allowed to pass parameters.

Reply:
Is this something you had in mind? 95b93a70431422f71a852d245bc55fc3db30c628
It's kind-of ok for me, but introduces another problem: I had to modify many existing templates to use the macro and also pass `action_node_properties=action_node_properties` to each of them, which is not too nice...
Maybe you have a simpler solution idea?

Reply:
Unfortunately not. A refactor is necessary, but it seems to me that it will not be difficult. 

From the beginning, I was afraid of using templates in this way, but I thought it would not happen.